quickwit/quickwit-indexing/src/actors/merge_pipeline.rs

```diff
@@ -88,15 +93,23 @@ impl Actor for MergePipeline {
 
 impl MergePipeline {
     pub fn new(params: MergePipelineParams) -> Self {
+        let (merge_planner_mailbox, merge_planner_inbox) =
+            create_mailbox::<MergePlanner>("MergePlanner".to_string(), QueueCapacity::Unbounded);
         Self {
             params,
             previous_generations_statistics: Default::default(),
             handles: None,
             kill_switch: KillSwitch::default(),
             statistics: MergeStatistics::default(),
+            merge_planner_inbox,
```

Comment:
this used to be in params and I just pulled it out.

It greatly simplifies the construction.